samples/dashboards/demo/project-management/demo.html

```diff
@@ -9,7 +9,24 @@
     <h2 class="dashboard-title" >Project management dashboard</h2>
     <p class="dashboard-description">Current progress in <b>week 22</b></p>
 </div>
-<div id="container"></div>
+<div id="container">
+    <div class="highcharts-dashboards-row">
```

Comment:
Should we use here classes like we have in the `dashboards.css` for example:
- `highcharts-dashboards-row`
- `highcharts-dashboards-cell`

They might conflict with them.

Reply:
We can reduce it to `cell` / `row`. I agree